langserver/server.go

```diff
@@ -164,6 +167,15 @@ func (s *server) connectPrometheus(url string) error {
 			return err
 		}
 
+		bodyBytes, _ := ioutil.ReadAll(resp.Body)
+		jsonResponse := map[string]map[string]string{}
+
+		err = json.Unmarshal(bodyBytes, &jsonResponse)
+
+		if err == nil {
+			s.PrometheusVersion = jsonResponse["data"]["version"]
+		}
+
```

Comment:
The should be a logging message (`s.client.ShowMessage`) when the version cannot be determined.